### PR TITLE
feat: add on-demand planner startup overview refresh

### LINKED ISSUE #97: Add in-session planner overview refresh via skills/runtime (no new operator command)
## Summary
Add an on-demand planner overview refresh that is usable **within an existing planner agent session** via skills/runtime flow, not as a new `atelier` operator command.

## Clarified rule
- `atelier` CLI commands are for human operators.
- Atelier-managed agents do not call `atelier` directly; they use skills/runtime access (including Beads reads).

## Problem
Planner startup provides useful context (messages + queue + epics), but there is no single in-session skill/runtime action to regenerate that same overview later during an active planner session.

## Requested behavior
Provide a planner-session overview refresh that can be invoked during an active planner session and outputs:
- unread planner inbox messages,
- queued messages,
- active epic list in stable `epic_list --show-drafts` format.

## Constraints
- No new `atelier` operator command for this feature.
- Read-only behavior: no claiming/mutation during overview refresh.
- Stable deterministic output for before/after comparisons.
- Reuse existing helper logic where possible.

## Acceptance criteria
- Existing planner session can invoke one skill/runtime action to reproduce startup overview.
- Output includes unread inbox, queue, and draft-inclusive epic listing.
- No state mutation side effects.
- Output format documented and test-covered.

### PR DESCRIPTION
**Pull request by shaug**
## Summary
- add an in-session planner overview refresh via skill/runtime flow (no new `atelier` operator command)
- print unread inbox messages, queued messages (queue name + claim state), and active epics in stable `epic_list --show-drafts` format
- reuse shared epic overview rendering to keep planner overview output aligned with epic listing behavior
- document the in-session refresh behavior and add tests for empty and populated overview cases

## Acceptance Criteria
- [x] One in-session planner skill/runtime action reproduces startup overview at any time in session
- [x] Output includes unread inbox, queued messages, and `epic_list --show-drafts` content
- [x] Overview refresh is read-only and performs no claims/mutations
- [x] Output format is deterministic and documented
- [x] Tests cover no-message and populated scenarios

## Testing
- `just format`
- `just lint`
- `env -u ATELIER_AGENT_ID -u ATELIER_SESSION_KEY just test`

## Tickets
- Fixes #97

### CODE REVIEW

**Comment by shaug** [comment]
Scope correction for #97:

1. Remove the new operator command surface (`atelier plan-overview`) and related CLI wiring/docs.
2. Implement this as a planner in-session skill/runtime action only (invokable by an already-running planner agent).
3. Preserve read-only behavior and deterministic output (inbox + queue + `epic_list --show-drafts`).
4. Keep agent rule explicit: atelier-managed agents should not call `atelier`; operators do.

Issue #97 has been updated to capture this scope precisely. Please retarget implementation and acceptance criteria to the updated issue definition.

File: src/atelier/commands/plan.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
     git,
     hooks,
     paths,
+    planner_overview,
```
Comment (shaug) [inline src/atelier/commands/plan.py]: this is no longer part of the desired functionality for this ticket
